BankSDK/GiniBankSDKExample/GiniBankSDKExampleTests/ScreenAPICoordinator+UI_Only.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ScreenAPICoordinator+UI_Only.swift
+//  GiniBankSDKExampleTests
+//
+//  Created by Nadya Karaban on 28.03.22.
+//
+
+import XCTest
+@testable import GiniBankSDKExample
+@testable import GiniBankAPILibrary
+@testable import GiniBankSDK
+@testable import GiniCaptureSDK
+
+class ScreenAPICoordinator___UI_Only: XCTestCase {
+        
+    let client = Client(id: "",
+                            secret: "",
+                            domain: "")
+    var screenAPICoordinator = ScreenAPIUIOnlyCoordinator()
+    
+    lazy var configuration: GiniBankConfiguration = {
+        let configuration = GiniBankConfiguration()
+        configuration.fileImportSupportedTypes = .pdf_and_images
+        configuration.openWithEnabled = true
+        configuration.qrCodeScanningEnabled = true
+        configuration.multipageEnabled = true
+        configuration.flashToggleEnabled = true
+        configuration.navigationBarItemTintColor = .white
+       return configuration
+    }()
+
+    func testInitialization() {
+        _ = GiniBank.viewController(withDelegate: screenAPICoordinator, withConfiguration: configuration)
```

Comment:
The two other updated public API methods should be tested as well:
* `GiniBank.viewController(... withTrackingDelegate:)`
* `GiniBank.setConfiguration(...)`

Reply:
A great catch 😇.